config/ironic.conf.j2

```diff
@@ -128,6 +128,10 @@ use_ipmitool_retries = true
 # interval of Y in between each tries.
 min_command_interval = 5
 command_retry_timeout = 60
+# List of possible cipher suites versions that can be
+# supported by the hardware in case the field `cipher_suite`
+# is not set for the node. (list value)
+cipher_suite_versions = ['3', '17']
```

Comment:
I haven't seen this syntax in conf files, are you sure it works? usually it should be 

```ini
cipher_suite_versions = 3,17
```

Reply:
I confirmed, the Python syntax does not work:
```
2021-06-09 16:58:46.653 2005081 DEBUG oslo_service.service [-] ipmi.cipher_suite_versions     = ["['3'", "'17']"] log_opt_values /home/dtantsur/Projects/ironic/.tox/venv/lib/python3.9/site-packages/oslo_config/cfg.py:2609
```